utilities/kml2geojson.py

```diff
@@ -67,7 +67,7 @@ def parse_kml(filename):
           coordinates.append([float(fm.group(1)), float(fm.group(2))])
         elif(re.search("</coordinates>", line)):
           if(ismulti):
-            multicoords[idname].append(coordinates)
+            multicoords[idname].append([coordinates])
```

Comment:
this was writing out MultiPolygons incorrectly, though the geojson renderer didn't complain. Fixed.